src/public/js/cardModal.js

```diff
@@ -0,0 +1,100 @@
+(function($) {
+    //this file should be imported into the board handlebars file
+    //this variable should be the <div> that the modal is going to be inserted in to
+    let placeModalHere = $("#placeModalHere");
```

Comment:
We'll get rid of this section of the code when we Frankenstein this functionality into board-page